src/Plugin/views/display/Eva.php

```diff
@@ -186,12 +186,12 @@ public function buildOptionsForm(&$form, FormStateInterface $form_state) {
   }
 
   public function validateOptionsForm(&$form, FormStateInterface $form_state) {
+    parent::validateOptionsForm($form, $form_state);
     switch ($form_state->get('section')) {
```

Comment:
Can I get you to either put the `break` back or change this to an `if` statement?  Just want to try to prevent logic mistakes down the road.